tests/framework/vm_config_missing_vcpu_mem_size_mib.json

```diff
@@ -0,0 +1,30 @@
+{
+  "boot-source": {
+    "kernel_image_path": "vmlinux.bin",
+    "boot_args": "console=ttyS0 reboot=k panic=1 pci=off",
+    "initrd_path": null
+  },
+  "drives": [
+    {
+      "drive_id": "rootfs",
+      "path_on_host": "bionic.rootfs.ext4",
+      "is_root_device": true,
+      "partuuid": null,
+      "is_read_only": false,
+      "cache_type": "Unsafe",
+      "io_engine": "Sync",
+      "rate_limiter": null
+    }
+  ],
+  "machine-config": {
+    "vcpu_count": 2,
+    "smt": false,
+    "track_dirty_pages": false
+  },
+  "balloon": null,
+  "network-interfaces": [],
+  "vsock": null,
+  "logger": null,
+  "metrics": null,
+  "mmds-config": null
```

Comment:
nit: all of those null entries are not mandatory, right? So we may remove them to keep the file more focused
Same for specific configs like the drive cache_type

Reply:
ok, I have minimised the json configs.